drivers/nrf9160_gps/nrf9160_gps.c

```diff
@@ -158,6 +179,27 @@ static void gps_thread(int dev_ptr)
 
 		switch (raw_gps_data.data_id) {
 		case NRF_GNSS_PVT_DATA_ID:
+			if (gps_is_blocked(&raw_gps_data.pvt)) {
+				if (operation_blocked) {
+					/* Avoid spamming the logs. */
+					continue;
+				}
+
+				LOG_INF("Waiting for time window to operate");
```

Comment:
I think this information belongs in the .rst for this driver or in the doxygen documentation and not here.

Reply:
I think it's useful debugging feedback for users runtime, but I could remove it. Thoughts, @Rallare ?

Reply:
How about keeping this line at `DBG` level to let the user know why the GPS isn't operating and give them a chance to manually enable (e)DRX or PSM, but convert the two following lines to comments?

Reply:
I took the liberty to go ahead and push my proposed changes for now. Keeping this one line as debug log message, and converting the rest to comments.